### PR TITLE
Improve TripCard

### PR DESCRIPTION
**Pull request by Bidjii**

### CODE REVIEW

File: src/tripCard/index.tsx [hunk 1]
```diff
@@ -75,14 +76,27 @@ const TripCard = ({
...
+          </Text>
         </div>
         <div className="kirk-tripCard-bottom">
           <div className="kirk-tripCard-driver">
-            <Avatar image={driver.avatarUrl} alt={driver.firstName} />
-            <span>{driver.firstName}</span>
+            <div className="kirk-tripCard-avatar">
+              <Avatar
+                image={driver.avatarUrl}
+                alt={driver.firstName}
```
Comment (Tuxosaurus) [inline src/tripCard/index.tsx]: nit: not sure the explicit `alt` is needed since the image is followed by the same text

Comment (PerrineBoissieres) [reply src/tripCard/index.tsx]: It's not needed, unless you want to precise more information than the driver's name.

File: src/tripCard/style.ts [hunk 2]
```diff
@@ -43,23 +36,21 @@ export default css`
...
-    line-height: ${font.m.lineHeight};
+    margin-right: ${space.l};
   }
 
-  .kirk-tripCard :global(.kirk-tripCard-driver > div) {
-    /* Adding the width of the step point to get the right alignment */
-    margin-right: calc(${space.xl} + 6px);
+  /* To match the width of the time area of the Itinerary */
+  :global(.kirk-tripCard-avatar) {
+    min-width: 64px;
```
Comment (PerrineBoissieres) [inline src/tripCard/style.ts]: Create a variable for those kind of numbers ;)

File: src/tripCard/index.tsx (hunk 1 above)
Comment (Tuxosaurus) [reply src/tripCard/index.tsx]: Yes that's what I meant, here the text is the same.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,16 +1,22 @@
 # Unreleased
+
 - **[NEW]** ItemInfo component
 - **[NEW]** Icons for Navigation (BankIcon, BellIcon, BubbleIcon, CoinIcon, CrossDiscIcon, HomeIcon, NewspaperIcon, ProfileIcon, TicketIcon)
 - **[UPDATE]** `IncreaseIcon` and `DecreaseIcon` are now `PlusIcon` and `MinusIcon`
+- **[UPDATE]** `TripCard` is now using `Text` component
+- **[FIX]** Behavior of the bottom of the `TripCard`
```
Comment (pith) [inline CHANGELOG.md]: > Behavior of the bottom of the `TripCard`

it's not really explicit what is fixed

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,7 +4,7 @@
 - **[NEW]** Icons for Navigation (BankIcon, BellIcon, BubbleIcon, CoinIcon, CrossDiscIcon, HomeIcon, NewspaperIcon, ProfileIcon, TicketIcon)
 - **[UPDATE]** `IncreaseIcon` and `DecreaseIcon` are now `PlusIcon` and `MinusIcon`
 - **[UPDATE]** `TripCard` is now using `Text` component
-- **[FIX]** Behavior of the bottom of the `TripCard`
+- **[FIX]** Margin between the elements when resizing on the bottom part of the `TripCard`
```
[the remark this answers is not in the document]
Comment (Bidjii) [reply CHANGELOG.md]: Will change it when releasing new version, won't ask for another round of approval :)

**Pull request merged**